adodb.inc.php

```diff
@@ -1681,7 +1681,14 @@ function GetAll($sql, $inputarr=false) {
 		return $arr;
 	}
 
-	function GetAssoc($sql, $inputarr=false,$force_array = false, $first2cols = false) {
+		/**
+		 * @param string $sql
+		 * @param false|array $inputarr
+		 * @param bool $force_array
+		 * @param bool $first2cols
+		 * @return false|array
+		 */
```

Comment:
remove extra indent, PHPDoc's opening `/**` should be aligned with *function* keyword